runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineConfiguration.java

```diff
@@ -85,7 +85,7 @@ public class EngineConfiguration extends Configuration
                 EngineConfiguration::decodeHostResolver, EngineConfiguration::defaultHostResolver);
         ENGINE_BUDGETS_BUFFER_CAPACITY = config.property("budgets.buffer.capacity", 1024 * 1024);
         ENGINE_LOAD_BUFFER_CAPACITY = config.property("load.buffer.capacity", 1024 * 8);
-        ENGINE_STREAMS_BUFFER_CAPACITY = config.property("streams.buffer.capacity", 1024 * 1024);
+        ENGINE_STREAMS_BUFFER_CAPACITY = config.property("streams.buffer.capacity",  2097152);
```

Comment:
Please don't change the defaults if avoidable.

If a specific IT requires a different value for this configuration then please override it for just that IT.